source/_components/notify.telegram.markdown

```diff
@@ -45,22 +46,51 @@ $ python3
 To enable Telegram notifications in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
+# Example configuration.yaml entry for the Telegram Bot
+telegram_bot:
+  platform: webhooks
```

Comment:
I don't think we should assume that a user has exposed the install to the outside, which is required for webhooks. Better to suggest the `polling` platform.